packages/core/src/components/drawer/drawer.tsx

```diff
@@ -66,14 +66,6 @@ export interface IDrawerProps extends OverlayableProps, IBackdropProps, Props {
      */
     position?: Position;
 
-    /**
-     * Whether the application should return focus to the last active element in the
-     * document after this drawer closes.
-     *
-     * @default true
-     */
-    shouldReturnFocusOnClose?: boolean;
```

Comment:
is this not a break?

Reply:
I think it's more like a QOL improvement for anyone that relies on keyboard navigation, since the focus would previously return to `body`. If you were manually setting focus in `onClose`, that still runs at the end of this change so your custom focus behavior will still work.

Reply:
it's not a break, `DrawerProps` extends from `OverlayableProps`!

Reply:
ah, i missed that! 👍